plugins/agents/process/process.go

```diff
@@ -0,0 +1,36 @@
+package process
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"github.com/gansoi/gansoi/plugins"
+	"github.com/gansoi/gansoi/transports"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	plugins.RegisterAgent("process", Process{})
+}
+
+// Process will check if a process with exact name is running on the host.
+type Process struct {
+	Name string `json:"name" description:"Exact process name"`
+}
+
+// RemoteCheck implements plugins.RemoteAgent.
+func (m *Process) RemoteCheck(transport transports.Transport, result plugins.AgentResult) error {
+
+	out, _, err := transport.Exec("pidof", m.Name)
+	if err != nil {
+		return errors.Wrap(err, "pidof")
+	}
+
+	resultAsBytes, err := ioutil.ReadAll(out)
+	if err != nil {
+		return err
+	}
+	resultAsStr := string(resultAsBytes)
+	result.AddValue("Running", len(strings.Fields(resultAsStr)))
```

Comment:
How about not casting to `string` and simply using `bytes.Fields()`? Maybe renaming `resultAsBytes` to `result`.

Reply:
I just realized the `result` identifier is already used. Well, you'll figure it out ;-)